lib/libc/stdio.c

```diff
@@ -6,16 +6,26 @@
  * https://opensource.org/licenses/MIT
  */
 #include <lk/debug.h>
+#include <lk/err.h>
 #include <stdio.h>
 #include <stdarg.h>
 #include <string.h>
+#include <stdlib.h>
 #include <sys/types.h>
 #include <platform/debug.h>
 
+#ifdef WITH_LIB_FS
```

Comment:
Can probably optimize this a bit to just wrap around the .use_fs line to avoid as much copying of code.